packages/amplify-codegen-appsync-model-plugin/src/utils/get-type-info.ts

```diff
@@ -12,7 +12,6 @@ export function getTypeInfo(typeNode: TypeNode, schema: GraphQLSchema): TypeInfo
     return {
       ...getTypeInfo(typeNode.type.type, schema),
       isList: true,
-      isListNullable: false,
```

Comment:
Can we be explicit in here by setting this to false

Reply:
Yeah, I'll update that... I didn't do this explicitly in the past since it increased the number of test cases that needed modification.